spec/draft/profile-spec.md

```diff
@@ -340,6 +340,85 @@ model User {
 }
 ```
 
+# Example
+
+Example : example Identifier { ExampleInput ExampleOutput }
```

Comment:
One last thing - is the example name required? Previously it wasn't and parser doesn't require it, but it's a minimal change.

Reply:
Let's match it up with the parser for now. I'll make it optional.